spec/requests/swagger_spec.rb

```diff
@@ -3436,7 +3415,38 @@
       subject.untested_mappings.delete('/v0/sign_in/callback')
       subject.untested_mappings.delete('/v0/sign_in/logout')
 
+      subject.untested_mappings.delete('/v0/post911_gi_bill_status')
```

Comment:
this entry will be deleted once we remove the endpoint and update swagger documentation